### PR TITLE
Gutenberg/activate preformat block

### PR DESCRIPTION
**Pull request by marecar3**
Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1264

GB mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1615

To test:

Follow test instructions on gb-mobile PR: wordpress-mobile/gutenberg-mobile#1615

PR submission checklist:

- [x] I have considered adding unit tests where possible.

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java [hunk 1]
```diff
@@ -2061,7 +2061,7 @@ private static String toHtml(SpannableStringBuilder builder,
         clearMetaSpans(builder);
         parser.syncVisualNewlinesOfBlockElements(builder);
         Format.postProcessSpannedText(builder, true);
-        return EndOfBufferMarkerAdder.removeEndOfTextMarker(parser.toHtml(builder, false));
+        return EndOfBufferMarkerAdder.removeEndOfTextMarker(parser.toHtml(builder, false, false));
```
Comment (hypest) [inline libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java]: I think we can use the `@JvmOverloads` annotation in the Aztec method in Kotlin to allow the Java side to call the method without the new extra parameter). Check out [this answer on SO](https://stackoverflow.com/a/34518024).

Comment (marecar3) [reply libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java]: Thanks for the hint. 
I have added annotation and now we don't need to pass extra param to the Kotlin method.

**Comment by marecar3** [comment]
Hey @hypest, I think that I have addressed all issues so this one is ready for another iteration of review.

**Comment by hypest** [comment]
I've LGTM'd the PR but, let's update the Aztec hash to a tag before we merge.

**Comment by hypest** [comment]
It seems that the failing Firebase tests are not related to the work on this PR and happening on other commits merged to develop already so, we'll not fix them as part of this PR and merge it.

**Pull request merged**